sql/moz-fx-data-shared-prod/fenix_derived/feature_usage_events_v1/query.sql

```diff
@@ -0,0 +1,1471 @@
+    ) AS sync_auth_recovered_users
+--sync_auth_scan_pairing
+    ,
+    SUM(sync_auth_scan_pairing) AS sync_auth_scan_pairing,
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN sync_auth_scan_pairing > 0
+          THEN client_id
+      END
+    ) AS sync_auth_scan_pairing_users
+--sync_auth_sign_in
+    ,
+    SUM(sync_auth_sign_in) AS sync_auth_sign_in,
+    COUNT(DISTINCT CASE WHEN sync_auth_sign_in > 0 THEN client_id END) AS sync_auth_sign_in_users
+--sync_auth_sign_out
+    ,
+    SUM(sync_auth_sign_out) AS sync_auth_sign_out,
+    COUNT(DISTINCT CASE WHEN sync_auth_sign_out > 0 THEN client_id END) AS sync_auth_sign_out_users
+--sync_auth_sign_up
+    ,
+    SUM(sync_auth_sign_up) AS sync_auth_sign_up,
+    COUNT(DISTINCT CASE WHEN sync_auth_sign_up > 0 THEN client_id END) AS sync_auth_sign_up_users
+--sync_auth_use_email
+    ,
+    SUM(sync_auth_use_email) AS sync_auth_use_email,
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN sync_auth_use_email > 0
+          THEN client_id
+      END
+    ) AS sync_auth_use_email_users
+--sync_auth_use_email_problem
+    ,
+    SUM(sync_auth_use_email_problem) AS sync_auth_use_email_problem,
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN sync_auth_use_email_problem > 0
+          THEN client_id
+      END
+    ) AS sync_auth_use_email_problem_users
+/*Privacy*/
+--hp_private_mode_tapped
+    ,
+    SUM(hp_private_mode_tapped) AS hp_private_mode_tapped,
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN hp_private_mode_tapped > 0
+          THEN client_id
+      END
+    ) AS hp_private_mode_tapped_users
+--tab_tray_private_mode_switched
+    ,
+    SUM(tab_tray_private_mode_switched) AS tab_tray_private_mode_switched,
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN tab_tray_private_mode_switched > 0
+          THEN client_id
+      END
+    ) AS tab_tray_private_mode_switched_users
+--app_icon_private_tab_tapped
+    ,
+    SUM(app_icon_private_tab_tapped) AS app_icon_private_tab_tapped,
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN app_icon_private_tab_tapped > 0
+          THEN client_id
+      END
+    ) AS app_icon_private_tab_tapped_users
+--tab_tray_private_mode_tapped
+    ,
+    SUM(tab_tray_private_mode_tapped) AS tab_tray_private_mode_tapped,
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN tab_tray_private_mode_tapped > 0
+          THEN client_id
+      END
+    ) AS tab_tray_private_mode_tapped_users
+--etp_setting_changed
+    ,
+    SUM(etp_setting_changed) AS etp_setting_changed,
+    COUNT(
+      DISTINCT
+      CASE
+        WHEN etp_setting_changed > 0
+          THEN client_id
+      END
+    ) AS etp_setting_changed_users
+--etp_settings
+    ,
+    SUM(etp_settings) AS etp_settings,
+    COUNT(DISTINCT CASE WHEN etp_settings > 0 THEN client_id END) AS etp_settings_users
+--etp_shield
+    ,
+    SUM(etp_shield) AS etp_shield,
+    COUNT(DISTINCT CASE WHEN etp_shield > 0 THEN client_id END) AS etp_shield_users
+--etp_tracker_list
+    ,
```

Comment:
Could we just move the comma to be at the end of the previous statement rather than on a new line by itself?